src/include/storage/storage_structure/lists/list_handle.h

```diff
@@ -0,0 +1,128 @@
+#pragma once
+
+#include <cstdint>
```

Comment:
I don't think we need to inclde cstdint in this file. I see types.h is using the uint32_t, uint64_t. Since list_handle.h includes types.h, it should be able to use uint32_t,64_t directly.

Reply:
Done.